indexer/packages/v4-proto-parser/src/order-helpers.ts

```diff
@@ -23,3 +23,10 @@ export function isStatefulOrder(orderFlag: number | String): boolean {
   }
   return numberOrderFlag === ORDER_FLAG_CONDITIONAL || numberOrderFlag === ORDER_FLAG_LONG_TERM;
 }
+
+export function isIOC(tif: IndexerOrder_TimeInForce): boolean {
```

Comment:
why does isIOC return true if the order if fill or kill? Do we want to differentiate or is it because fok is a subset of ioc?

Reply:
Updated to `requiresImmediateExecution` matching the naming of the same [method](https://github.com/dydxprotocol/v4-chain/blob/ece92b4216ce61c6e28090124aae1c4f1144ec2f/protocol/x/clob/types/order.go#L156) in the protocol.